From f942efe73ab17798d23a8e09884338561a24c41f Mon Sep 17 00:00:00 2001
From: Colin Ian King <colin.king@canonical.com>
Date: Tue, 22 Aug 2017 13:51:08 +0100
Subject: [PATCH] staging: r8822be: fix a couple of spelling mistakes
Git-commit: f942efe73ab17798d23a8e09884338561a24c41f
Patch-mainline: v4.14-rc1
References: FATE#326887

Trivial fixes to spelling mistake in pr_err error messages:

errorr -> error
mismactch -> mismatch

Signed-off-by: Colin Ian King <colin.king@canonical.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Acked-by: Takashi Iwai <tiwai@suse.de>

---
 .../rtlwifi/halmac/halmac_88xx/halmac_api_88xx.c |  2 +-
 .../halmac/halmac_88xx/halmac_func_88xx.c        | 16 ++++++++--------
 2 files changed, 9 insertions(+), 9 deletions(-)

diff --git a/drivers/staging/rtlwifi/halmac/halmac_88xx/halmac_api_88xx.c b/drivers/staging/rtlwifi/halmac/halmac_88xx/halmac_api_88xx.c
index fadb35f3ba88..ee7c3d0a6e34 100644
--- a/drivers/staging/rtlwifi/halmac/halmac_88xx/halmac_api_88xx.c
+++ b/drivers/staging/rtlwifi/halmac/halmac_88xx/halmac_api_88xx.c
@@ -1256,7 +1256,7 @@ halmac_init_mac_cfg_88xx(struct halmac_adapter *halmac_adapter,
 
 	status = halmac_api->halmac_init_trx_cfg(halmac_adapter, mode);
 	if (status != HALMAC_RET_SUCCESS) {
-		pr_err("halmac_init_trx_cfg errorr = %x\n", status);
+		pr_err("halmac_init_trx_cfg error = %x\n", status);
 		return status;
 	}
 	status = halmac_api->halmac_init_protocol_cfg(halmac_adapter);
diff --git a/drivers/staging/rtlwifi/halmac/halmac_88xx/halmac_func_88xx.c b/drivers/staging/rtlwifi/halmac/halmac_88xx/halmac_func_88xx.c
index cc39679aaa64..50f6f256795a 100644
--- a/drivers/staging/rtlwifi/halmac/halmac_88xx/halmac_func_88xx.c
+++ b/drivers/staging/rtlwifi/halmac/halmac_88xx/halmac_func_88xx.c
@@ -2477,7 +2477,7 @@ halmac_parse_psd_data_88xx(struct halmac_adapter *halmac_adapter, u8 *c2h_buf,
 			"[TRACE]Seq num : h2c -> %d c2h -> %d\n",
 			psd_set->seq_num, h2c_seq);
 	if (h2c_seq != psd_set->seq_num) {
-		pr_err("[ERR]Seq num mismactch : h2c -> %d c2h -> %d\n",
+		pr_err("[ERR]Seq num mismatch : h2c -> %d c2h -> %d\n",
 		       psd_set->seq_num, h2c_seq);
 		return HALMAC_RET_SUCCESS;
 	}
@@ -2531,7 +2531,7 @@ halmac_parse_efuse_data_88xx(struct halmac_adapter *halmac_adapter, u8 *c2h_buf,
 			halmac_adapter->halmac_state.efuse_state_set.seq_num,
 			h2c_seq);
 	if (h2c_seq != halmac_adapter->halmac_state.efuse_state_set.seq_num) {
-		pr_err("[ERR]Seq num mismactch : h2c -> %d c2h -> %d\n",
+		pr_err("[ERR]Seq num mismatch : h2c -> %d c2h -> %d\n",
 		       halmac_adapter->halmac_state.efuse_state_set.seq_num,
 		       h2c_seq);
 		return HALMAC_RET_SUCCESS;
@@ -2728,7 +2728,7 @@ halmac_parse_h2c_ack_phy_efuse_88xx(struct halmac_adapter *halmac_adapter,
 			halmac_adapter->halmac_state.efuse_state_set.seq_num,
 			h2c_seq);
 	if (h2c_seq != halmac_adapter->halmac_state.efuse_state_set.seq_num) {
-		pr_err("[ERR]Seq num mismactch : h2c -> %d c2h -> %d\n",
+		pr_err("[ERR]Seq num mismatch : h2c -> %d c2h -> %d\n",
 		       halmac_adapter->halmac_state.efuse_state_set.seq_num,
 		       h2c_seq);
 		return HALMAC_RET_SUCCESS;
@@ -2765,7 +2765,7 @@ halmac_parse_h2c_ack_cfg_para_88xx(struct halmac_adapter *halmac_adapter,
 			h2c_seq);
 	if (h2c_seq !=
 	    halmac_adapter->halmac_state.cfg_para_state_set.seq_num) {
-		pr_err("Seq num mismactch : h2c -> %d c2h -> %d\n",
+		pr_err("Seq num mismatch : h2c -> %d c2h -> %d\n",
 		       halmac_adapter->halmac_state.cfg_para_state_set.seq_num,
 		       h2c_seq);
 		return HALMAC_RET_SUCCESS;
@@ -2833,7 +2833,7 @@ halmac_parse_h2c_ack_update_packet_88xx(struct halmac_adapter *halmac_adapter,
 			halmac_adapter->halmac_state.update_packet_set.seq_num,
 			h2c_seq);
 	if (h2c_seq != halmac_adapter->halmac_state.update_packet_set.seq_num) {
-		pr_err("[ERR]Seq num mismactch : h2c -> %d c2h -> %d\n",
+		pr_err("[ERR]Seq num mismatch : h2c -> %d c2h -> %d\n",
 		       halmac_adapter->halmac_state.update_packet_set.seq_num,
 		       h2c_seq);
 		return HALMAC_RET_SUCCESS;
@@ -2916,7 +2916,7 @@ halmac_parse_h2c_ack_channel_switch_88xx(struct halmac_adapter *halmac_adapter,
 			halmac_adapter->halmac_state.scan_state_set.seq_num,
 			h2c_seq);
 	if (h2c_seq != halmac_adapter->halmac_state.scan_state_set.seq_num) {
-		pr_err("[ERR]Seq num mismactch : h2c -> %d c2h -> %d\n",
+		pr_err("[ERR]Seq num misactch : h2c -> %d c2h -> %d\n",
 		       halmac_adapter->halmac_state.scan_state_set.seq_num,
 		       h2c_seq);
 		return HALMAC_RET_SUCCESS;
@@ -2968,7 +2968,7 @@ halmac_parse_h2c_ack_iqk_88xx(struct halmac_adapter *halmac_adapter,
 			"[TRACE]Seq num : h2c -> %d c2h -> %d\n",
 			halmac_adapter->halmac_state.iqk_set.seq_num, h2c_seq);
 	if (h2c_seq != halmac_adapter->halmac_state.iqk_set.seq_num) {
-		pr_err("[ERR]Seq num mismactch : h2c -> %d c2h -> %d\n",
+		pr_err("[ERR]Seq num misactch : h2c -> %d c2h -> %d\n",
 		       halmac_adapter->halmac_state.iqk_set.seq_num, h2c_seq);
 		return HALMAC_RET_SUCCESS;
 	}
@@ -3018,7 +3018,7 @@ halmac_parse_h2c_ack_power_tracking_88xx(struct halmac_adapter *halmac_adapter,
 			h2c_seq);
 	if (h2c_seq !=
 	    halmac_adapter->halmac_state.power_tracking_set.seq_num) {
-		pr_err("[ERR]Seq num mismactch : h2c -> %d c2h -> %d\n",
+		pr_err("[ERR]Seq num mismatch : h2c -> %d c2h -> %d\n",
 		       halmac_adapter->halmac_state.power_tracking_set.seq_num,
 		       h2c_seq);
 		return HALMAC_RET_SUCCESS;
-- 
2.19.1

